### PR TITLE
Added piezoClass to fix launching of Cavity Fault Display

### PR DESCRIPTION
**Pull request by Derikka**

### CODE REVIEW

File: cavityFaultDisplay.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
 class CavityFaultDisplay(Display):
     def __init__(self, parent=None, args=None, macros=None):
-        super().__init__(parent=parent, args=args,
-                         ui_filename="frontend/cavityfaultdisplay.ui",
-                         macros=macros)
+        super(CavityFaultDisplay, self).__init__(parent=parent, args=args,
```
Comment (lisazacarias) [inline cavityFaultDisplay.py]: Did it no longer work without these parameters? I feel like it was working before...

**Review by lisazacarias — APPROVED** [review]

File: cavityFaultDisplay.py (hunk 1 above)
Comment (Derikka) [reply cavityFaultDisplay.py]: Oh, maybe! I was using Zack's code as a reference and I just blindly copied as much as possible from his setup. Let me try deleting and see what happens

Comment (Derikka) [reply cavityFaultDisplay.py]: Those parameters were indeed unnecessary, so they are now deleted!

File: cavityFaultDisplay.py [hunk 2]
```diff
@@ -74,34 +66,52 @@ def __init__(self, parent=None, args=None, macros=None):
...
 
-    @staticmethod
-    def statusLabelCallback(statuslabel: QLabel, codelabel: QLabel, fault: Fault, **kw):
-        try:
-            if fault.isFaulted():
-                statuslabel.setText("FAULTED")
-                statuslabel.setStyleSheet("background-color: rgb(255,0,0); font-weight: "
-                                          "bold; border: 2px solid black; color: white;")
-                codelabel.setStyleSheet("font-weight:bold;")
+class EnumLabel(PyDMLabel):
```
Comment (lisazacarias) [inline cavityFaultDisplay.py:80]: I love this!!!!

**Review by lisazacarias — APPROVED** [review]
I assume this has been tested so I'm approving! Well done!!!

**Pull request merged**